mrbgems/mruby-socket/test/unix.rb

```diff
@@ -1,5 +1,3 @@
-unless SocketTest.win?
```

Comment:
Is this change intentional? Does UNIXSocket test run OK for Windows?

Reply:
At least many `mruby-io` related stuff is failing too so I don't think that line will do anything good.
(Unless you want to hide the CI test errors in windows.)
You should exclude whole `mruby-io` and `mruby-socket` mrbgem instead of doing those kind of thing.

This PR only makes fix to exceptions inside Windows' dynamic library, that's all.
I've fixed segv error and there is the rest errors.

Reply:
I suggest to merge this since CI is failing (crash) on Windows.

![crash](https://go-gyazo.appspot.com/67707d82905466d1.png)